### PR TITLE
Feature/rpnv1 440 node cli

### PR DESCRIPTION
**Pull request by flotothemoon**

### CODE REVIEW

File: radixdlt-cli/build.gradle [hunk 1]
```diff
@@ -30,14 +30,16 @@ buildscript {
 apply plugin: 'java'
 apply plugin: 'com.github.johnrengelman.shadow'
 apply plugin: 'groovy'
+apply plugin: 'application'
 repositories {
-    jcenter()
-    maven { url "https://oss.sonatype.org/content/repositories/snapshots" }
-    maven { url 'https://jitpack.io' }
+	jcenter()
```
Comment (clstrfsck) [inline radixdlt-cli/build.gradle]: Should use spaces instead of tabs here for consistency.

Comment (clstrfsck) [reply radixdlt-cli/build.gradle]: Actually, it doesn't look like any of this actually changed, so should be reverted.

File: radixdlt-cli/build.gradle [hunk 2]
```diff
@@ -30,14 +30,16 @@ buildscript {
...
+	maven { url "https://oss.sonatype.org/content/repositories/snapshots" }
+	maven { url 'https://jitpack.io' }
 }
 
+mainClassName = 'com.radixdlt.cli.RadixCLI'
 compileJava {
-    options.compilerArgs += ["-Aproject=${project.group}/${project.name}"]
+	options.compilerArgs += ["-Aproject=${project.group}/${project.name}"]
```
Comment (clstrfsck) [inline radixdlt-cli/build.gradle]: Revert unnecessary changes.

File: radixdlt-cli/src/main/groovy/com/radixdlt/cli/ShowValidatorConfig.groovy [hunk 3]
```diff
@@ -33,18 +34,25 @@ class ShowValidatorConfig implements Runnable {
 	@CommandLine.ArgGroup(exclusive = true, multiplicity = "0..1")
 	Composite.IdentityInfo identityInfo
 
-	void run() {
+	@CommandLine.Option(names = ["-d", "--address"], paramLabel = "ADDRESS", description = "Address to which message is sent", required = false)
```
Comment (clstrfsck) [inline radixdlt-cli/src/main/groovy/com/radixdlt/cli/ShowValidatorConfig.groovy]: The description for this option doesn't seem very apt now.

**Review by clstrfsck — CHANGES_REQUESTED** [review]
Please update the command-line address description, and there is some minor whitespace issues that would be good to clean up too.

File: radixdlt-cli/src/main/groovy/com/radixdlt/cli/ShowValidatorConfig.groovy (hunk 3 above)
Comment (flotothemoon) [reply radixdlt-cli/src/main/groovy/com/radixdlt/cli/ShowValidatorConfig.groovy]: Whoops, yes, thanks.

**Review by clstrfsck — APPROVED** [review]
